drivers/timer/nrf_rtc_timer.c

```diff
@@ -552,11 +558,12 @@ void sys_clock_disable(void)
 
 	irq_disable(NRF5_IRQ_RTC1_IRQn);
 
-	SYS_CLOCK_RTC->EVTENCLR = RTC_EVTENCLR_COMPARE0_Msk;
-	SYS_CLOCK_RTC->INTENCLR = RTC_INTENCLR_COMPARE0_Msk;
+	nrf_rtc_event_disable(SYS_CLOCK_RTC, RTC_EVTENSET_COMPARE0_Msk);
+	nrf_rtc_int_disable(SYS_CLOCK_RTC, RTC_INTENSET_COMPARE0_Msk);
```

Comment:
Why change CLR to SET msk?

Reply:
No good reason apart from copy paste laziness and that they both are the same bit.
It is indeed better to use the CLR ones. So fixed it.